recipe/patches/0022-gcc_13.patch

```diff
@@ -0,0 +1,37 @@
+Description: add missing <cstdint> includes to fix build with GCC 13
+Author: Dmitry Shachnev <mitya57@debian.org>
+Forwarded: https://github.com/mapbox/mapbox-gl-native/pull/16669
+Last-Update: 2023-06-18
+
+--- a/src/3rdparty/mapbox-gl-native/include/mbgl/util/geometry.hpp
```

Comment:
You need to add `qtlocation` here before the path to adapt to the full qt tarball we are using here: `a/qtlocation/src/3rdparty/mapbox-gl-native/include/mbgl/util/geometry.hpp`.

Reply:
sigh you definitely nerd snippped me to restart the job.